### ISSUE TITLE
Running the CMS workload from LCG_10X

### ISSUE THREAD

**Issue by sciaba** [iris-hep/analysis-grand-challenge#160]
To be able to run the workload from SWAN using Dask to scale out to HTCondor, ideally all dependencies should be satisfied by a given LCG configuration, as the remote Dask workers will have that. Would it make sense to create the minimal requirements.txt to be used on top of the latest LCG configuration (103) to enable this use case?

**Comment by matthewfeickert** [comment]
> Ideally all dependencies should be satisfied by a given LCG configuration

I don't think this is a reasonable thing to try to do. LCG views are (in my view) for Python a bit of a nightmare as they become environments that are not well defined in terms of requirements and fragile to build upon.

While I'm sure this is possible to do, I think it would just be painful and lead to lots of headaches compared to just fully defining the environment.
